opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -118,255 +145,122 @@ message MetricDescriptor {
-
-  // value itself.
-  double value = 4;
-}
+    // NONNEGATIVE is the set of numbers greater than or equal to zero.
+    NONNEGATIVE = 1;
 
-// HistogramDataPoint is a single data point in a timeseries that describes the time-varying
-// values of a Histogram. A Histogram contains summary statistics for a population of values,
-// it may optionally contain the distribution of those values across a set of buckets.
-message HistogramDataPoint {
-  // The set of labels that uniquely identify this timeseries.
-  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
-
-  // start_time_unix_nano is the time when the cumulative value was reset to zero.
-  //
-  // The cumulative value is over the time interval (start_time_unix_nano, time_unix_nano].
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
-  //
-  // Value of 0 indicates that the timestamp is unspecified. In that case the timestamp
-  // may be decided by the backend.
-  // Note: this field is always unspecified and ignored if MetricDescriptor.type==GAUGE_HISTOGRAM.
-  fixed64 start_time_unix_nano = 2;
-
-  // time_unix_nano is the moment when this value was recorded.
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
-  fixed64 time_unix_nano = 3;
-
-  // count is the number of values in the population. Must be non-negative. This value
-  // must be equal to the sum of the "count" fields in buckets if a histogram is provided.
-  uint64 count = 4;
-
-  // sum of the values in the population. If count is zero then this field
-  // must be zero. This value must be equal to the sum of the "sum" fields in buckets if
-  // a histogram is provided.
-  double sum = 5;
-
-  // Bucket contains values for a bucket.
-  message Bucket {
-    // The number of values in each bucket of the histogram, as described by
-    // bucket_options.
-    uint64 count = 1;
-
-    // Exemplars are example points that may be used to annotate aggregated
-    // Histogram values. They are metadata that gives information about a
-    // particular value added to a Histogram bucket.
-    message Exemplar {
-      // Value of the exemplar point. It determines which bucket the exemplar belongs to.
-      // If bucket_options define bounds for this bucket then this value must be within
-      // the defined bounds.
-      double value = 1;
-
-      // time_unix_nano is the moment when this exemplar was recorded.
-      // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
-      fixed64 time_unix_nano = 2;
-
-      // exemplar_attachments are contextual information about the example value.
-      // Keys in this list must be unique.
-      repeated opentelemetry.proto.common.v1.StringKeyValue attachments = 3;
-    }
-
-    // exemplar is an optional representative value of the bucket.
-    Exemplar exemplar = 2;
+    // NONPOSITIVE is the set of numbers less than or equal to zero.
+    NONPOSITIVE  = 2;
```

Comment:
Could be used if measuring your enjoyment of a global pandemic :trollface:.

Yeah, it was just added for completeness. Do you think we should just remove it?

Reply:
I think if we're going to put the refinements (instead of e.g. the instrument names) in the proto then it makes sense to start small, and only include the refinements for the instruments we expect to support. Assuming it's easier to add these later than remove them.

Reply:
I would also vote for removing things that we are not sure about usefulness.

Reply:
removed.